e2e-tests/functions

```diff
@@ -1449,29 +1449,52 @@ start_cluster() {
 
 get_latest_restorable_time() {
 	local cluster=$1
-	local timestamp
-	local timeout
+	local first_timestamp
+	local seconf_timestamp
+	local retry=0
 
-	until [[ $timestamp != "" && $timestamp != "null" ]]; do
+	until [[ $first_timestamp != "" && $first_timestamp != "null" && $first_timestamp == $second_timestamp ]]; do
```

Comment:
Please, add a comment about why we need `first_timestamp` and `second_timestamp`, (as we spoke in the chat), because many people scratch their heads wondering about that.

Reply:
done